cmd/node/config/api.toml

```diff
@@ -1,3 +1,13 @@
+# Logging holds settings related to api requests logging
+[Logging]
+    # LoggingEnabled - if this flag is set to true, then if a requests exceeds a threshold or it is unsuccessful, then
+    # a log will be printed
+    LoggingEnabled = false
+
+    # ThresholdNanoSeconds represents the maximum duration to consider a request as normal. Above this, if the LoggingEnabled
+    # flag is set to true, then a log will be printed
+    ThresholdInNanoSeconds = 1
```

Comment:
isn't this too low? Suggestion to change this to microseconds as defining time duration in nanoseconds will yield too large values. And maybe change this to 100 microseconds?

Reply:
done